python/ml_wrappers/model/model_wrapper.py

```diff
@@ -38,7 +39,7 @@
     module_logger.debug('Could not import torch, required if using a PyTorch model')
 
 
-def wrap_model(model, examples, model_task=ModelTask.UNKNOWN, num_classes=None):
+def wrap_model(model, examples, model_task=ModelTask.UNKNOWN, classes=None, num_classes=None):
```

Comment:
```suggestion
def wrap_model(model, examples, model_task: str = ModelTask.UNKNOWN, classes: Union[list, np.array] = None, num_classes: int = None):
```
Nit: perhaps something like this if this works.